pkgs/applications/graphics/inkscape/extensions/textext-paths.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/textext/base.py b/textext/base.py
+index 48506ee..c28d0ba 100644
+--- a/textext/base.py
++++ b/textext/base.py
+@@ -125,6 +125,8 @@ class TexText(inkex.EffectExtension):
+     def __init__(self):
+ 
+         self.config = Settings("config.json")
```

Comment:
Won’t this already contain the values, as passed to `setup.py`? Or can this be a user config? Would be nice to clarify in a comment.

Reply:
I'm slightly on thin ice here, but indeed I'm pretty sure that's how it works. Added a comment.